meta.js

```diff
@@ -51,15 +51,15 @@ module.exports = {
       "message": "Port that mock server listen at",
       "default": 7878
     },
-    "i18n": {
```

Comment:
Because in the Vue files a variable name must be  i18n.

<script>
 import i18n from 'locales/things'
 export default {
   locales,
   i18n,
....


Therefore, the conditional translation variable was renamed to not coincide with the variable in the vue files

Reply:
The variables `i18n` in vue file are not wrapped with `{{}}`, so it does not affect these variables I think.

Reply:
OK. I'm new (only a month) to node js.